ush/tropcy_relocate.sh

```diff
@@ -433,7 +429,7 @@ fi
 
 MP_PULSE=0
 MP_TIMEOUT=600
-GDATE10=$( ${NDATE:?} -06 ${CDATE10})
+GDATE10=$(date --utc +%Y%m%d%H -d "${CDATE10} - 06 hours")
```

Comment:
I don't think this is not correct
```suggestion
GDATE10=$(date --utc +%Y%m%d%H -d "${CDATE10:0:8} ${CDATE10:8:2} - 6 hours")
```
because
```
~ ❯❯❯ CDATE10="2024061218"                                                                                                   ⏎
~ ❯❯❯ gdate --utc +%Y%m%d%H -d "${CDATE10} - 06 hours"
202406121718
~ ❯❯❯ gdate --utc +%Y%m%d%H -d "${CDATE10:0:8} ${CDATE10:8:2} - 06 hours"
2024061212
```
The `0` in `06` is unnecessary

Reply:
Thanks for the fix!